include/contact_widgets.php

```diff
@@ -135,8 +135,8 @@ function fileas_widget($baseurl,$selected = '') {
 
 	$matches = false;
 	$terms = array();
-    $cnt = preg_match_all('/\[(.*?)\]/',$saved,$matches,PREG_SET_ORDER);
-    if($cnt) {
+	$cnt = preg_match_all('/\[(.*?)\]/',$saved,$matches,PREG_SET_ORDER);
```

Comment:
Standards: Could you please add a space after each comma in the parameter list? ☺️